typescript/libs/global_vars.py

```diff
@@ -2,13 +2,14 @@
 import re
 
 import sublime
+from os.path import dirname
 from .logger import *
 
 
 # get the directory path to this file; 
-LIBS_DIR = os.path.dirname(os.path.abspath(__file__))
-PLUGIN_DIR = os.path.dirname(LIBS_DIR)
-PACKAGES_DIR = os.path.dirname(PLUGIN_DIR)
+LIBS_DIR = dirname(dirname(os.path.abspath(__file__)))
+PLUGIN_DIR = dirname(LIBS_DIR)
+PACKAGES_DIR = dirname(PLUGIN_DIR)
```

Comment:
Correct, the Sublime APIs are available immediately on ST2, but not until plugin_loaded on ST3.  I used a pattern at https://github.com/billti/BillsSTP/blob/master/init.py#L14 which worked pretty well, but doing work in a init function that is called on load in ST2, and in plugin_loaded on ST3.

Generally we want to avoid work in module loading anyway where possible as a best practice.

On a meta-note, if this path is not reliable, what are we using it for?  Do we need it at all?